packages/unit-testing/stubs/datasetLink/datasetLink.js

```diff
@@ -1,4 +1,4 @@
-define(['./DatasetLinkInstance'], function (DatasetLink) {
+define(['../workbook/Expression', './DatasetLinkInstance'], function (DatasetLink) {
```

Comment:
Missing Expression parameter. dataset.DatasetInstance should be also injected as Dataset parameter.